cypress.config.ts

```diff
@@ -0,0 +1,40 @@
+import { defineConfig } from "cypress";
+import prisma from './lib/prisma'
+require('dotenv').config()
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false,
```

Comment:
`supportFile` is set to false to remove the support folder with `e2e.ts` and `command.ts`